src/bin2header.cpp

```diff
@@ -17,7 +17,7 @@
 using namespace std;
 
 
-const string version = "0.2.0";
+const string version = "0.2.1";
```

Comment:
Please don't update hardcoded version information with your changes. I will do this when I am ready to create release.